input/syslog.js

```diff
@@ -1,26 +1,29 @@
-const syslogd = require('syslogd')
+const syslogParse = require('syslog-parse')
+
 const { fromJS } = require('immutable')
 
+const socket = require('./socket')
```

Comment:
`socket` is not an input so it should not be in the `input` directory.

Reply:
@benfle it actually is, and I'm planning to document it

Reply:
Nice, I missed that.